pkg/driver/node/envprovider/provider.go

```diff
@@ -0,0 +1,90 @@
+// Package envprovider provides utilities for accessing environment variables to pass Mountpoint.
+package envprovider
+
+import (
+	"fmt"
+	"os"
+	"slices"
+)
+
+const (
+	EnvRegion                = "AWS_REGION"
+	EnvDefaultRegion         = "AWS_DEFAULT_REGION"
+	EnvSTSRegionalEndpoints  = "AWS_STS_REGIONAL_ENDPOINTS"
+	EnvMaxAttempts           = "AWS_MAX_ATTEMPTS"
+	EnvProfile               = "AWS_PROFILE"
+	EnvConfigFile            = "AWS_CONFIG_FILE"
+	EnvSharedCredentialsFile = "AWS_SHARED_CREDENTIALS_FILE"
+	EnvRoleARN               = "AWS_ROLE_ARN"
+	EnvWebIdentityTokenFile  = "AWS_WEB_IDENTITY_TOKEN_FILE"
+	EnvEC2MetadataDisabled   = "AWS_EC2_METADATA_DISABLED"
+	EnvAccessKeyID           = "AWS_ACCESS_KEY_ID"
+	EnvSecretAccessKey       = "AWS_SECRET_ACCESS_KEY"
+	EnvSessionToken          = "AWS_SESSION_TOKEN"
+
+	EnvMountpointCacheKey = "UNSTABLE_MOUNTPOINT_CACHE_KEY"
```

Comment:
Nit: Not aligned

Reply:
Addressed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/362/commits/83d852bc785f51635dd8072772fe72713cfaba78. gofmt formats that way if there is an empty line between blocks, I wanted to separate this one from others as it's unstable, but it indeed seemed odd. Now it should be fine.